klippy/extras/adxl345.py

```diff
@@ -135,11 +136,33 @@ def __init__(self, config):
         gcode.register_mux_command("ACCELEROMETER_QUERY", "CHIP", self.name,
                                    self.cmd_ACCELEROMETER_QUERY,
                                    desc=self.cmd_ACCELEROMETER_QUERY_help)
+        gcode.register_mux_command("READ_ADXL345", "CHIP", self.name,
+                                   self.cmd_ADXL345_DEBUG_READ,
+                                   desc=self.cmd_ADXL345_DEBUG_READ_help)
+        gcode.register_mux_command("SET_ADXL345", "CHIP", self.name,
+                                   self.cmd_ADXL345_DEBUG_WRITE,
+                                   desc=self.cmd_ADXL345_DEBUG_WRITE_help)
```

Comment:
Typo?  Registers `READ_ADXL345` when documentation uses `ADXL345_DEBUG_READ`?

Reply:
Ah, that's a good catch, I was renaming the command to a more appropriate one, and I guess I forgot to rename these instances. Renamed.